src/ck-core/ckrdma.C

```diff
@@ -1602,69 +1609,10 @@ void CkReplaceSourcePtrsInBcastMsg(envelope *env, NcpyBcastInterimAckInfo *bcast
 #endif
 }
 
-#if CMK_SMP
-void updatePeerCounterAndPush(envelope *env) {
-  int pe;
-  int numops, rootNode;
-
-  PUP::toMem p((void *)(((CkMarshallMsg *)EnvToUsr(env))->msgBuf));
-  PUP::fromMem up((void *)((CkMarshallMsg *)EnvToUsr(env))->msgBuf);
-
-  up|numops;
-  up|rootNode;
-  p|numops;
-  p|rootNode;
-
-  CkNcpyBuffer source;
-
-  up|source;
-
-  pe = CmiNodeFirst(CmiMyNode());
-
-  void *ref = (void *)source.refAckInfo;
-  NcpyBcastRecvPeerAckInfo *peerAckInfo = (NcpyBcastRecvPeerAckInfo *)ref;
-
-  p|source;
-  CmiSpanningTreeInfo &t = *(getSpanningTreeInfo(rootNode));
-  if(peerAckInfo->decNumPeers() - 1 == 0) {
-    QdCreate(1);
-    CMI_ZC_MSGTYPE(env) = CMK_ZC_BCAST_RECV_ALL_DONE_MSG;
-    CmiPushPE(CmiRankOf(peerAckInfo->peerParentPe), env);
-  }
-}
 
+#if CMK_SMP || CMK_UTH_VERSION
```

Comment:
`CMK_UTH_VERSION` is no longer part of Charm

Reply:
Fixed.